### PR TITLE
HPCC-16884 Fix core gathering stats from parallel loop graph

### PR DESCRIPTION
**Pull request by ghalliday**
Regression introduced by HPCC-16517

Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

### CODE REVIEW

**Comment by ghalliday** [comment]
@richardkchapman please review

File: roxie/ccd/ccdcontext.cpp [hunk 1]
```diff
@@ -1470,14 +1470,19 @@ class CRoxieContextBase : implements IRoxieSlaveContext, implements ICodeContext
...
-            graph.clear();
-            childGraphs.kill();
+            cleanupGraphs();
             graphStats.clear();
             if (error)
                 throw error;
         }
     }
 
+    void cleanupGraphs()
```
Comment (richardkchapman) [inline roxie/ccd/ccdcontext.cpp]: Changes in this file seem somewhat unrelated...
